configure.ac

```
@@ -1,5 +1,5 @@
 AC_PREREQ([2.63])
-AC_INIT([eos-chrome-plugin-updater], [1.0.0], [cosimo@endlessm.com], [eos-chrome-plugin-updater], [http://www.endlessm.com/])
+AC_INIT([eos-chrome-plugin-updater], [1.0.1], [cosimo@endlessm.com], [eos-chrome-plugin-updater], [http://www.endlessm.com/])
```

While we're here,

```suggestion
AC_INIT([eos-chrome-plugin-updater], [1.0.1], [emdev@endlessm.com], [eos-chrome-plugin-updater], [http://www.endlessm.com/])
```

though I wouldn't normally bother bumping the version number, personally, since the git revision will be appended when pushing into OBS, and presumably this means you need to mess around with `Version_` tags etc.

---

I kept the bump as I believe the changes are worth enough of a new version - also re-pushed the tags to make CI happy.